identity/handler_test.go

```diff
@@ -566,6 +567,33 @@ func TestHandler(t *testing.T) {
 		})
 	})
 
+	t.Run("suite=PATCH identities", func(t *testing.T) {
```

Comment:
Please improve this test. We need to know that all fields are imported and processed correctly. Especially things like verified and recovery addresses and credentials.

Reply:
I vastly expanded the test code now. PTAL :)